tests/integration/zipkin_integration_test.py

```diff
@@ -175,8 +175,8 @@ def check_spans(spans):
         assert nested_span.binary_annotations[0].key == 'nested_key'
         assert nested_span.binary_annotations[0].value == 'nested_value'
         # Local nested spans report timestamp and duration
-        assert nested_span.timestamp == 100 * USECS
-        assert nested_span.duration == 200 * USECS
+        # assert nested_span.timestamp == 100 * USECS
+        # assert nested_span.duration == 200 * USECS
```

Comment:
So, that one's still there since it's a "feature" that I'm dropping in this change and I couldn't decide whether it's something useful or not.

Specifically before if you passed in custom `cr` & `cs` annotations, py_zipkin would use those to generate duration and timestamp ignoring the automatically generated ones. In the new version that's no longer true. That means `timestamp` and `duration` won't match with `cr`, `cs` anymore.

The other issue is that setting annotations and assuming that'll change the timestamp and duration is a very weird way imo and won't work in V2. Maybe I should just add 2 optional `duration` and `timestamp` arguments to `zipkin_span` and log a deprecation warning if people set those annotations.